controllers/storagecluster/exporter.go

```diff
@@ -709,3 +714,76 @@ func createMetricsExporterRolebindings(ctx context.Context, r *StorageClusterRec
 
 	return nil
 }
+
+func createRookCephClusterRolebindings(ctx context.Context,
+	r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
+	// rookCephMonitorMgrRoleBinding is a cluster rolebinding for monitor mgr
+	rookCephMonitorMgrRoleBinding := rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rook-ceph-monitor-mgr",
+			Namespace: instance.Namespace,
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "rook-ceph-monitor-mgr",
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "rook-ceph-mgr",
+				Namespace: instance.Namespace,
```

Comment:
I found that this ServiceAccount only exists in OperatorNamespace. So, binding is required in that namespace only. Let's change this so we don't provide unwanted access.

Reply:
Done